src/components/menu/_menu.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @ngdoc module
+ * @name material.components.menu
+ */
+
+// TODO(s)
+//   - enable `width` attribute and set width based on spec (http://www.google.com/design/spec/components/menus.html#menus-specs)
+
+angular.module('material.components.menu', [
+  'material.core',
+  'material.components.backdrop'
+])
+.directive('mdMenu', MenuDirective);
+
+function MenuDirective($mdMenu) {
+  return {
+    restrict: 'E',
+    require: 'mdMenu',
+    controller: function() { }, // empty function to be built by link
```

Comment:
I need a new `fn` because otherwise I'd pollute `angular.noop` with a bunch of properties in the link function below